ios/RNIterableAPI/ReactIterableAPI.swift

```diff
@@ -257,6 +261,12 @@ class ReactIterableAPI: RCTEventEmitter {
         IterableAPI.updateEmail(email, onSuccess: nil, onFailure: nil)
     }
     
+    @objc(handleUniversalLink:resolver:rejecter:)
+    func handleUniversalLink(link: String, resolver: RCTPromiseResolveBlock, rejecter: RCTPromiseRejectBlock) {
```

Comment:
Since this is Swift we can name the method as below and this will not have any effect on javascript etc.
`handle(universalLink urlString: String, resolver...)`

Reply:
This way we can all this method as `handle(universalLink: "https://somewhere.com/a/xyz" .... )`
Just a suggestion. I don't feel strongly about this. I think this would be more consistent with Swift naming.